GitUI/Translation/Czech.xlf

```diff
@@ -6977,7 +6977,7 @@ Do you want to continue?</source>
         
       </trans-unit>
       <trans-unit id="createABranchOnThisCommitToolStripMenuItem.Text">
-        <source>Create a branch on this commit</source>
+        <source>Create a branch on this commit...</source>
```

Comment:
This should be ellipsis character

Reply:
@gerhardol apologies for not replying until now. We took a long weekend in the wilderness and I'm just getting back online now.

This PR has been merged but I still wanted to investigate this idea across the solution.

A quick grep across the codebase suggests there are many instances of three periods `...`, especially in `*.Designer.cs` files:

```text
$ grep -r "\.\.\." --include "*.Designer.cs" | wc -l
95
$ grep -r "…" --include "*.Designer.cs" | wc -l
0
$ grep -r "…" --include "*.cs" | wc -l
6
$ grep -r "\.\.\." --include "*.xlf" | wc -l
31944
$ grep -r "…" --include "*.xlf" | wc -l
128
```

There is actually a visual difference between the two forms when rendered:

![image](https://user-images.githubusercontent.com/350947/44023949-cd505494-9ee3-11e8-97c8-034ab0a45c1b.png)

Here are some examples from other applications.

Visual Studio menus: 
![image](https://user-images.githubusercontent.com/350947/44024209-945708a8-9ee4-11e8-88d3-e35ed200cc68.png)

Firefox menus: 
![image](https://user-images.githubusercontent.com/350947/44024227-9d826df0-9ee4-11e8-8009-f77975216a80.png)

Chrome menus: 
![image](https://user-images.githubusercontent.com/350947/44024249-aca5de02-9ee4-11e8-8120-5035482d4958.png)

It looks like most applications use the three dot form.

An ellipsis character would likely force non-ASCII encoding for a few source files. This probably isn't a problem in practice.

Do you know of some additional reason to use the ellipsis character here?

Reply:
Ellipsis has shorter space between the periods, I say at least Visual Studio and Firefox use ellipsis. (Should be possible to find for Firefox and Chromium at least.)

This is a matter of style, what is recommended, differs also within a certain language, also within e.g. American English. I guess many use periods as it is easier to type and they do not know about ellipsis (I have ignored ellipsis for years myself). 
Anyway, ellipsis and periods should not be mixed. Better to align on one standard.

I believe ellipsis should have been used here. For other occurrences, as well as this PR as the PR is merged, the that is a separate issue. All in one PR?

Reply:
Another option is to amend strings loaded from translation files. If they end in `...` we sub in the ellipsis. This has failure modes, in theory. They're less likely than translators not knowing to/how to enter ellipsis characters.